helpdesk_mgmt_activity/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,7 @@
+The module adds the following features:
+
+- Refer a ticket to the Odoo model record
```

Comment:
- Link a ticket to an Odoo model record
- Set the list of available models for a Helpdesk team
- Pre-configure ticket description template based on it's category
- Create an activity for the linked record right from the Ticket
- Change the Ticket's stage based on the activity state